dll/win32/setupapi/cfgmgr.c

```diff
@@ -1075,7 +1075,8 @@ CM_Add_Range(
     }
     else
     {
-
+        HeapFree(GetProcessHeap(), 0, pRange);
+        UNIMPLEMENTED;
```

Comment:
Oh, and since this is the unimplemented path, could you also add after the "UNIMPLEMENTED;" the following: `ret = CR_CALL_NOT_IMPLEMENTED;` ?

Reply:
That's a behavior change that could cause regressions. I'd consider this out of scope.

Reply:
Hmmm OK then...